Formula/t/tcpreplay.rb

```diff
@@ -24,6 +24,10 @@ class Tcpreplay < Formula
 
   uses_from_macos "libpcap"
 
+  # Fix argument order in macro call.
+  # TODO: Upstream this.
+  patch :DATA
```

Comment:
Ah, it looks like there is an incompatibility between the bundled `lib/queue.h` header and the system `sys/queue.h` header.